### DISCUSSION TITLE
Grouping multiple traces from the same session and creating a thread.

### DISCUSSION THREAD

**Discussion by knownbymanoj** [Arize-ai/phoenix#2892] [Q&A] [answered]
Can someone give me more information about how to group traces with same session id and view them as threads? I found this functionality in langsmith.
Thanks in advance.

**Comment by axiomofjoy** [comment]
Hey @knownbymanoj, sessions are on the [roadmap](https://github.com/orgs/Arize-ai/projects/45/views/1), although not part of our current milestone.

**Comment by axiomofjoy** [comment, marked as answer]
You can follow [this ticket](https://github.com/Arize-ai/phoenix/issues/2619) for updates. Thanks for your patience. We know sessions are important and will incorporate them as soon as we can.
